shared/oae/api/oae.api.util.js

```diff
@@ -1087,7 +1087,8 @@ define(['exports', 'require', 'jquery', 'underscore', 'oae.api.config', 'markdow
 
                     // If there are no search results and the query is an email address, present an
                     // option for the user to have the email address as an option
-                    if (options.allowEmail && isValidEmail && _.isEmpty(data.results)) {
+                    var allowInvitingGuests = configAPI.getValue('oae-tenants', 'guests', 'allow');
+                    if (options.allowEmail && isValidEmail && _.isEmpty(data.results) && allowInvitingGuests) {
```

Comment:
Should `allowInvitingGuests` be tied into `options.allowEmail` somehow, so it takes care of things like showing the `__MSG__NO_RESULTS_FOUND__` message ?